nixos/tests/sanoid.nix

```
@@ -38,7 +38,7 @@ in {
 
       services.syncoid = {
         enable = true;
-        sshKey = "/root/.ssh/id_ecdsa";
+        sshKey = "/tmp/id_ecdsa";
```

Because people sometimes copy configuration from tests:

```suggestion
        sshKey = "/var/lib/syncoid/id_ecdsa";
```

---

`/tmp/` is not a good location to put fixed filenames.